Source/WebCore/svg/graphics/SVGImage.h

```diff
@@ -44,7 +44,7 @@ class Settings;
 class SVGImage final : public Image {
 public:
     static Ref<SVGImage> create(ImageObserver* observer) { return adoptRef(*new SVGImage(observer)); }
-    WEBCORE_EXPORT static RefPtr<SVGImage> tryCreateFromData(std::span<const uint8_t>);
+    WEBCORE_EXPORT static void tryCreateFromDataAndWaitForSubresources(std::span<const uint8_t>, CompletionHandler<void(RefPtr<SVGImage>&&)>&&);
```

Comment:
Do we need "wait for subresources" in the function name?

Reply:
Why do we need to "spread awareness"? It seems that having only the function that waits for subresources already does the job; hard to do it wrong unless we have a "do it wrong" function. I think the name implies the opposite, that it's reasonable to create an SVG image from data without waiting for subresources.

Reply:
I don't insist on changing the name. Just wanted to make my case for why not to!

Reply:
We do have an alternative way of getting an `SVGImage` instance unfortunately. But I'm also persuaded that this could be confusing as well so I'll shorten before landing.